packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts

```diff
@@ -32,17 +32,7 @@ export const actionCreationFull = {
     refundAddress,
     salt,
   },
-  version,
-};
-export const actionCreationFullSubsequent = {
-  action: RequestLogicTypes.ACTION_NAME.CREATE,
-  id: ExtensionTypes.ID.PAYMENT_NETWORK_ERC777_STREAM,
-  parameters: {
-    masterRequestId,
-    previousRequestId,
-    recurrenceNumber,
-  },
-  version,
+  version: '0.1.0',
```

Comment:
It looks like you're not using this code above, you may be able to clean it:
https://github.com/RequestNetwork/requestNetwork/blob/eb6d39d0db93ec53c9d61d7e2dd16b3da44ce831/packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts#L18-L20